pkg/mcp/server/server.go

```diff
@@ -455,11 +455,11 @@ func (con *connection) receive() {
 		if err != nil {
 			code := status.Code(err)
 			if code == codes.Canceled || err == io.EOF {
-				scope.Infof("MCP: connection %v: TERMINATED %q", con, err)
+				scope.Infof("MCP: connection %s: TERMINATED %v", con.peerAddr, err)
```

Comment:
It would be better to make this change to the connection's [`String()`](https://github.com/istio/istio/blob/80f7ced8f636201bdd70fb9d8da0244cf67e4ac2/pkg/mcp/server/server.go#L422) method instead of changing all of the call sites.

Reply:
I think we just need s/%v/%s since we have `String()`